src/modules/commander/commander_params.c

```diff
@@ -615,14 +615,28 @@ PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
  * Specify modes in which RC loss is ignored and the failsafe action not triggered.
  *
  * @min 0
- * @max 31
+ * @max 7
  * @bit 0 Mission
  * @bit 1 Hold
  * @bit 2 Offboard
  * @group Commander
  */
 PARAM_DEFINE_INT32(COM_RCL_EXCEPT, 0);
 
+/**
```

Comment:
The undocumented feature does not exist. Added https://github.com/PX4/PX4-Autopilot/pull/24652.